lasagne/layers/recurrent.py

```diff
@@ -757,3 +762,306 @@ def step_masked(input_dot_W_n, mask_n, cell_previous, hid_previous,
         self.cell_out = cell_out
 
         return hid_out
+
+
+class GRULayer(Layer):
+    """Gated Recurrent Layer (GRU)
+
+    Layer with gated recurrent units implemented as in [1]_, [2]_.
+
+    Parameters
+    ----------
+    incoming : a :class:`lasagne.layers.Layer` instance or a tuple
+        The layer feeding into this layer, or the expected input shape.
+    num_units : int
+        Number of hidden units.
+    W_in_to_resetgate : function or np.ndarray or theano.shared
```

Comment:
You have arguments to the function without any documentation, you need to do e.g.
```
W_in_to_resetgate : Theano shared variable, numpy array or callable
    Initializer for input-to-reset gate weight matrix.
```